{"uid":"b1de95a94747ac08","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"d8696551571014a71a874c85cadaf3fe","time":{"start":1702689158557,"stop":1702689168763,"duration":10206},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x555ec0905d33 <unknown>\n#1 0x555ec05c2f87 <unknown>\n#2 0x555ec0612c0e <unknown>\n#3 0x555ec0612d11 <unknown>\n#4 0x555ec065e004 <unknown>\n#5 0x555ec063c36d <unknown>\n#6 0x555ec065b297 <unknown>\n#7 0x555ec063c0e3 <unknown>\n#8 0x555ec0604044 <unknown>\n#9 0x555ec060544e <unknown>\n#10 0x555ec08ca861 <unknown>\n#11 0x555ec08ce785 <unknown>\n#12 0x555ec08b8285 <unknown>\n#13 0x555ec08cf41f <unknown>\n#14 0x555ec089c20f <unknown>\n#15 0x555ec08f3028 <unknown>\n#16 0x555ec08f31f7 <unknown>\n#17 0x555ec0904ed4 <unknown>\n#18 0x7f4186494ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7eff1d3a9310>\nemail = 'nirt236@gmail.com', password = '123456'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"24b85cb6016bec91e931120d240c10fa\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7eff1d4656c0>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x555ec0905d33 <unknown>\nE       #1 0x555ec05c2f87 <unknown>\nE       #2 0x555ec0612c0e <unknown>\nE       #3 0x555ec0612d11 <unknown>\nE       #4 0x555ec065e004 <unknown>\nE       #5 0x555ec063c36d <unknown>\nE       #6 0x555ec065b297 <unknown>\nE       #7 0x555ec063c0e3 <unknown>\nE       #8 0x555ec0604044 <unknown>\nE       #9 0x555ec060544e <unknown>\nE       #10 0x555ec08ca861 <unknown>\nE       #11 0x555ec08ce785 <unknown>\nE       #12 0x555ec08b8285 <unknown>\nE       #13 0x555ec08cf41f <unknown>\nE       #14 0x555ec089c20f <unknown>\nE       #15 0x555ec08f3028 <unknown>\nE       #16 0x555ec08f31f7 <unknown>\nE       #17 0x555ec0904ed4 <unknown>\nE       #18 0x7f4186494ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"json_data","time":{"start":1702689131961,"stop":1702689131961,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1702689131961,"stop":1702689131961,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1702689131961,"stop":1702689131961,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x555ec0905d33 <unknown>\n#1 0x555ec05c2f87 <unknown>\n#2 0x555ec0612c0e <unknown>\n#3 0x555ec0612d11 <unknown>\n#4 0x555ec065e004 <unknown>\n#5 0x555ec063c36d <unknown>\n#6 0x555ec065b297 <unknown>\n#7 0x555ec063c0e3 <unknown>\n#8 0x555ec0604044 <unknown>\n#9 0x555ec060544e <unknown>\n#10 0x555ec08ca861 <unknown>\n#11 0x555ec08ce785 <unknown>\n#12 0x555ec08b8285 <unknown>\n#13 0x555ec08cf41f <unknown>\n#14 0x555ec089c20f <unknown>\n#15 0x555ec08f3028 <unknown>\n#16 0x555ec08f31f7 <unknown>\n#17 0x555ec0904ed4 <unknown>\n#18 0x7f4186494ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7eff1d3a9310>\nemail = 'nirt236@gmail.com', password = '123456'\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"24b85cb6016bec91e931120d240c10fa\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7eff1d4656c0>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x555ec0905d33 <unknown>\nE       #1 0x555ec05c2f87 <unknown>\nE       #2 0x555ec0612c0e <unknown>\nE       #3 0x555ec0612d11 <unknown>\nE       #4 0x555ec065e004 <unknown>\nE       #5 0x555ec063c36d <unknown>\nE       #6 0x555ec065b297 <unknown>\nE       #7 0x555ec063c0e3 <unknown>\nE       #8 0x555ec0604044 <unknown>\nE       #9 0x555ec060544e <unknown>\nE       #10 0x555ec08ca861 <unknown>\nE       #11 0x555ec08ce785 <unknown>\nE       #12 0x555ec08b8285 <unknown>\nE       #13 0x555ec08cf41f <unknown>\nE       #14 0x555ec089c20f <unknown>\nE       #15 0x555ec08f3028 <unknown>\nE       #16 0x555ec08f31f7 <unknown>\nE       #17 0x555ec0904ed4 <unknown>\nE       #18 0x7f4186494ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1702689158557,"stop":1702689168762,"duration":10205},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x555ec0905d33 <unknown>\n#1 0x555ec05c2f87 <unknown>\n#2 0x555ec0612c0e <unknown>\n#3 0x555ec0612d11 <unknown>\n#4 0x555ec065e004 <unknown>\n#5 0x555ec063c36d <unknown>\n#6 0x555ec065b297 <unknown>\n#7 0x555ec063c0e3 <unknown>\n#8 0x555ec0604044 <unknown>\n#9 0x555ec060544e <unknown>\n#10 0x555ec08ca861 <unknown>\n#11 0x555ec08ce785 <unknown>\n#12 0x555ec08b8285 <unknown>\n#13 0x555ec08cf41f <unknown>\n#14 0x555ec089c20f <unknown>\n#15 0x555ec08f3028 <unknown>\n#16 0x555ec08f31f7 <unknown>\n#17 0x555ec0904ed4 <unknown>\n#18 0x7f4186494ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"e7cba981da18c5c","name":"public ip address","source":"e7cba981da18c5c.txt","type":"text/plain","size":13},{"uid":"a74a1b9899fdcb91","name":"Cookies","source":"a74a1b9899fdcb91.json","type":"application/json","size":3357},{"uid":"ccce74ec7bc2f962","name":"Session Storage","source":"ccce74ec7bc2f962.json","type":"application/json","size":358},{"uid":"e5df6fc1c37556a0","name":"Local Storage","source":"e5df6fc1c37556a0.json","type":"application/json","size":208},{"uid":"bea2cef11598701f","name":"Console Logs","source":"bea2cef11598701f.json","type":"application/json","size":2},{"uid":"63480b3340d5c68a","name":"Full Page Screenshot","source":"63480b3340d5c68a.png","type":"image/png","size":1276064},{"uid":"72a1f624e304cb14","name":"URL","source":"72a1f624e304cb14.uri","type":"text/uri-list","size":23},{"uid":"ed423395971640b9","name":"Network Logs","source":"ed423395971640b9.json","type":"application/json","size":25752}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"blocker"},{"name":"feature","value":"Login"},{"name":"epic","value":"Security"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1391-145"},{"name":"thread","value":"2375-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'nirt236@gmail.com'"},{"name":"password","value":"'123456'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"b1de95a94747ac08.json","parameterValues":["'nirt236@gmail.com'","'123456'"]}